server/index.ts

```diff
@@ -12,24 +16,27 @@ import { signInHandler, signUpHandler } from './handlers/userHandler';
 
   app.use(express.json());
 
-  const requestLoggerMiddleware: RequestHandler = (req, res, next) => {
-    console.log(req.method, req.path, '- body:', req.body);
-    next();
-  };
-
   app.use(requestLoggerMiddleware);
 
-  app.get('/v1/posts', asyncHandler(listPostsHandler));
-  app.post('/v1/posts', asyncHandler(createPostHandler));
+  app.get('/v1/posts/list', asyncHandler(listPostsHandler));
+  app.post('/v1/posts/new', asyncHandler(createPostHandler));
+  app.delete('/v1/posts/:id', asyncHandler(deletePostHandler));
+  app.get('/v1/posts/:id', asyncHandler(getPostHandler));
 
   app.post('/v1/signup', asyncHandler(signUpHandler));
   app.post('/v1/signin', asyncHandler(signInHandler));
-
-  const errHandler: ErrorRequestHandler = (err, req, res, next) => {
-    console.error('Uncaught exception:', err);
-    return res.status(500).send('Oops, an unexpected error occurred, please try again');
-  };
-  app.use(errHandler);
+  //TODO Create signOutHandler
```

Comment:
we likely won't need a sign out handler, but if we do we'll add it later when needed. we can remove the comment until we know we want it. :)

Reply:
okay.